tests/www/views/test_views_connection.py

```diff
@@ -180,3 +180,26 @@ def test_duplicate_connection_error(admin_client):
     assert resp.status_code == 200
     response = {conn[0] for conn in session.query(Connection.conn_id).all()}
     assert expected_result == response
+
+
+@pytest.fixture()
+def connection():
+    connection = Connection(
+        conn_id='conn1',
+        conn_type='Conn 1',
+        description='Conn 1 description',
+    )
+    with create_session() as session:
+        session.add(connection)
+    yield connection
+    with create_session() as session:
+        session.query(Connection).filter(Connection.conn_id == CONNECTION["conn_id"]).delete()
+    
+
```

Comment:
```suggestion


```

The test passes, but static checks are not happy. This should be the final fixup.